js/passwordShowStyle.js

```diff
@@ -0,0 +1,18 @@
+import { $ } from './common.js';
+
+const $iconEyeOpen = $('.icon-eye-open');
```

Comment:
icon은 뒤에 붙는게 좋을 것 같네요!

```suggestion
const $eyeOpenIcon = $('.eye-open-icon');
```